lib/uppy/ping/scheme.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Uppy.Ping do
+  use Ecto.Schema
+
+  alias Uppy.Job
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+
+  schema "pings" do
+    field :response, :string
+    field :time, :integer
```

Comment:
Would `duration` be a better name here?

Reply:
I was considering this, but `ping(8)` uses `time` as the name for this

Reply:
Actually time makes sense when I think of it in context of a ping.